mix.exs

```diff
@@ -38,7 +38,7 @@ defmodule Liblink.MixProject do
   defp deps do
     [
       {:jason, "~> 1.0"},
-      {:tesla, "~> 1.0.0"},
+      {:tesla, "~> 1.3.0"},
```

Comment:
Para não quebrar quem usa uma versão mais antiga do Tesla, melhor botar um range generoso, tipo `">= 1.0.0 and < 2.0.0"` ou similar.